src/components/Board/Board.tsx

```diff
@@ -23,9 +24,8 @@ class Board extends Component<IBoardProps> {
             className="paper"
             style={{backgroundColor: "lightgrey"}}
             onDragEnter={(e) => this.handleDragEnter(e)}
-            onDrop={(e) => this.handleDrop(e)}
-            onDragEnd={(e) => this.handleDragEnd(e)}
             onDragOver={(e) => this.handleDragOver(e)}
+            onDrop={(e) => this.handleDrop(e)}
```

Comment:
A small thing, but you can pass directly the function by referece
`onDrop={this.handleDrop}`

Reply:
done